code/modules/shuttle/shuttle.dm

```diff
@@ -320,17 +361,60 @@
 	if(smooth)
 		smooth_icon(src)
 
+/obj/docking_port/mobile/proc/jumpToNullSpace()
+	// Destroys the docking port and the shuttle contents.
+	// Not in a fancy way, it just ceases.
+
+	JUSTDOIT = TRUE //GOD WHY WHY GOD
```

Comment:
@Crazylemon64 when i do it it jumps it to null space properly but NOW it won't jump the template in with
ADMIN LOG: check_dock(): shuttle Snappop(tm)! cannot dock at the emergency centcom, error: someone_else_docked

so its doing it before an undock..or some shit..even when the old shuttles been jumped out.